client/src/AddVideo.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+function AddVideo({ videos, setVideos }) {
+  const [showValid, Setvalid] = useState(true);
+    const [newVideo, setNewVideo] = useState({});
+   const [videoTitle, setTitle] = useState("");
+  const [videoUrl, setUrl] = useState("");
+  //   const [, setVideos] = useState(videos);
+
+  const handleShowValid = () => {
+    Setvalid(!showValid);}
+  
+   function isValidUrl(url) {
+    try {
+      new URL(url);
+      return true;
+    } catch (error) {
+      return false;
+    }
+  }
+  const handleAddVideo = (event) => {
+    event.preventDefault();
+    videoTitle !== ""
+      ? (newVideo.title = videoTitle)
+      : alert("Please add a title for your video.");
```

Comment:
Probably better to use an `if` here instead; ternaries are used to assign a variable to two different variables conditionally, whereas you're using it to perform a side effect (causing an alert). This would set `videoTitle` to the result of calling `alert()` which will be `null`